libscript/sources/Parser.cxx

```diff
@@ -110,6 +112,11 @@ void Parser::tokens_from_file(const std::string &file, bool isContent, bool verb
         // get the line string clean of comments (anything after the first found hashtag is deleted)
         std::string line = remove_comments(text.substr(lineStart, lineEnd - lineStart));
 
+        // clean for -Number to (0 - Number)
+        std::regex Rpattern("(-(?=\\w+))(\\w+)");
+        std::string substitution = "(0 - $2)";
+        line = std::regex_replace(line, Rpattern, substitution);
+
```

Comment:
That is a hack, not a solution. Wouldn't it be better to have the interpreter to handle it properly ?

Reply:
While finding a solution would be better, it would require quite some rework to add this operator in. Since we are in a rush for the beta, and this hack only affects performance a tiny bit and only on the parser side, I think it is appropriate.